app/src/main/java/tec/ac/cr/marape/app/AddGuestActivity.kt

```diff
@@ -16,63 +17,65 @@ import tec.ac.cr.marape.app.adapter.UserView
 import tec.ac.cr.marape.app.model.Inventory
 import tec.ac.cr.marape.app.model.User
 
+const val ADDED_GUEST_INVENTORY = 3
+
 class AddGuestActivity : AppCompatActivity() {
+  private val db = FirebaseFirestore.getInstance()
   private val currentUserEmail: String = FirebaseAuth.getInstance().currentUser?.email ?: ""
-  private lateinit var userList: MutableList<User>
+  private lateinit var recyclerView: RecyclerView
+  private lateinit var searchUser: EditText
+  private var position: Int = 0
+  private lateinit var inventory: Inventory
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_add_guest)
 
-    val inventory = intent.getStringExtra("idInventory")
-    val searchUser: EditText = findViewById(R.id.entry_user_search);
-    val recyclerView: RecyclerView = findViewById(R.id.recycle_users);
-    recyclerView.layoutManager = LinearLayoutManager(this);
+    inventory = intent.getSerializableExtra("inventory") as Inventory
```

Comment:
Aquí el método `getSerializableExtra/1` está deprecado, es mejor utilizar `getSerializableExtra/2`. Este requiere la anotación `@RequiresApi(Build.VERSION_CODES.TIRAMISU)` esta se debe poner arriba del método `onCreate` (con esto me refiero a en la línea encima de `override fun onCreate(savedInstance: Bundle?)`

```suggestion
    inventory = intent.getSerializableExtra("inventory", Inventory::class.java)!!
```